docs/src/includes/_enums.md

```diff
@@ -638,6 +638,22 @@ Name | Data | Description
 ...check [particle_emitters.txt](game_data/particle_emitters.txt)... |  | 
 [MINIGAME_BROKENASTEROID_SMOKE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PARTICLEEMITTER.MINIGAME_BROKENASTEROID_SMOKE) | 219 | 
 
+## PAUSE
+
+
+> Search script examples for [PAUSE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE)
+
+Used in state.pause
+
+Name | Data | Description
+---- | ---- | -----------
+[MENU](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE.MENU) | 0x01 | Menu: Pauses the level timer and engine. Can't set, controller by the menu.<br/>
+[FADE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE.FADE) | 0x02 | Fade/Loading: Pauses all timers and engine.<br/>
+[CUTSCENE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE.CUTSCENE) | 0x04 | Cutscene: Pauses total/level time but not engine. Used by boss cutscenes.<br/>FLAG4:<br/>Unknown purpose: Pauses total/level time and engine. Might change this later!<br/>FLAG5:<br/>Unknown purpose: Pauses total/level time and engine. Might change this later!<br/>
+[FLAG4](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE.FLAG4) | 0x08 | 
+[FLAG5](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE.FLAG5) | 0x10 | 
```

Comment:
Something didn't work there with the comments?
the `FLAG5` is useful as it pauses everything but leaves one of the frame counter so set_global_interval works still (you seams to remove that info on the variable so maybe add it here?)